.travis.yml

```diff
@@ -5,6 +5,8 @@ cache:
   yarn: true
   directories:
   - node_modules
+env:
+  - NODE_ENV=test 
```

Comment:
Mind explaining why the tests were failing without this?

Reply:
@chrisberthe Tests were failing because of the `slate-analytic` package, it is trying to search for the `NODE_ENV` var, if it is not present then `slate-analytic` is going to prompt for an email just to have some analytics on Slate. It fails on Travis because Travis CI is waiting for the user's email input.

Reply:
That's interesting - we have a `SLATE_USER_EMAIL` env variable already set on Travis CI's side to bypass that prompt. This should have worked after you rebased from `master`.

Sorry for the extra work but would you mind removing this from `.travis.yml` and seeing if it still fails?

Reply:
@chrisberthe Oh, that's weird, I've rebased the branch with the upstream/master branch, no changes were detected since I did the same the last week, I removed the env var from the travis.yml file and the tests failed again because of the email prompt timeout.

I was looking for `SLATE_USER_EMAIL` in the code but I couldn't find anything.

Reply:
It's something you set [in Travis CI](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings), which we are currently doing for `SLATE_USER_EMAIL`. No idea why it's not fixing itself :')

We should technically see the following in the build but it's not there:

```
Setting environment variables from repository settings
$ export SLATE_USER_EMAIL=[secure]
```

^ This was grabbed from a previous PR that was merged.